### ISSUE TITLE
Document usability problems for screen readers

### ISSUE THREAD

**Issue by ragesoss** [WikiEducationFoundation/WikiEduDashboard#5298]
The Dashboard has some users — including an admin — who are blind and use screen readers.

Choose a screen reader from the following:
* VoiceOver on Mac OS
* VoiceOver on iOS
* JAWS on Windows (proprietary, costs money... but it might be possible to get a free education license)
* NVDA on Windows
* ORCA on Linux
* ChromeVox (Chrome browser plugin, cross-platform)

Use that screen reader (this may take some research and/or practice, to learn to use it), and navigate through either a Dashboard development environment or the live outreachdashboard.wmflabs.org site, and document any accessibility problems you find: pages that are harder than they should be to navigate via keyboard commands and screen reader nav tools, buttons and links that are not interpreted correctly via screen reader, and anything else that makes the Dashboard harder to use than it ought to be for screen reader users. You can either document problems in comments below, or open a new issue for each individual problem you find.

**Comment by RedWing0021** [comment]
Hello,
I would like to work on this issue. Please assign this to me if no one is working on it.

**Comment by ragesoss** [comment]
@RedWing0021 we don't assign issues, but you're welcome to work on this.

**Comment by AkankshaAI** [comment]
@ragesoss Since no is contributing to this issue ,I would like to contribute and work on this issue

**Comment by AkankshaAI** [comment]
Im learning to use it, using NVDA for windows but finding a bit of difficulty. Can you guide a bit for resources

**Comment by Aminehassou** [comment]
I added a possible fix to an accessibility issue. I tested using ChromeVox so I'm unsure if it works using other screen readers

**Comment by QuantumByteMaster** [comment]
@ragesoss  i would like to contribute on this issue . I will be using ORCA on linux.

**Comment by crocmons** [comment]
@ragesoss I would like to contribute to this issue . I will choose ChromeVox (Chrome browser plugin, cross-platform)

**Comment by Jay2006sawant** [comment]
@ragesoss I would like to contribute on this issue
